<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SailfishWizards::Internal::LocationSelect</class>
 <widget class="QWizardPage" name="SailfishWizards::Internal::LocationSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Name and location</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Class name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QLineEdit" name="pathEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="errorLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: red; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="hFileEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="nameHLabel">
       <property name="text">
        <string>Name header file:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="cppFileEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="nameCppLabel">
       <property name="text">
        <string>Name source file:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
